modules/dxp/apps/commerce-avalara/commerce-avalara-connector-web/src/main/java/com/liferay/commerce/avalara/connector/web/internal/util/AvalaraTaxCodesCommerceHealthHttpStatus.java

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.avalara.connector.web.internal.util;
+
+import com.liferay.commerce.avalara.connector.constants.CommerceAvalaraConstants;
+import com.liferay.commerce.avalara.connector.helper.CommerceAvalaraConnectorHelper;
+import com.liferay.commerce.constants.CommerceHealthStatusConstants;
+import com.liferay.commerce.health.status.CommerceHealthHttpStatus;
+import com.liferay.commerce.product.model.CPTaxCategory;
+import com.liferay.commerce.product.service.CPTaxCategoryLocalService;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.ServiceContextFactory;
+import com.liferay.portal.kernel.transaction.Propagation;
+import com.liferay.portal.kernel.transaction.TransactionConfig;
+import com.liferay.portal.kernel.transaction.TransactionInvokerUtil;
+import com.liferay.portal.kernel.util.ResourceBundleUtil;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+import java.util.concurrent.Callable;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Katie Nesterovich
+ */
+@Component(
+	enabled = false, immediate = true,
+	property = {
+		"commerce.health.status.display.order:Integer=70",
+		"commerce.health.status.key=" + CommerceAvalaraConstants.AVALARA_TAX_CODES_COMMERCE_HEALTH_STATUS_KEY
+	},
+	service = CommerceHealthHttpStatus.class
+)
+public class AvalaraTaxCodesCommerceHealthHttpStatus
```

Comment:
I think the name of this is ok, if we look at some of our other CommerceHealthHttpStatus classes we don't namespace them with Commerce, ex: PricingCommerceHealthHttpStatus, WishListContentCommerceHealthHttpStatus, etc.